concepts/pipeline/event_pipeline.md

```diff
@@ -0,0 +1,127 @@
+# Event Data Pipeline
+
+We collect event-oriented data from different sources. This data is collected and processed in a
+specific path through our data pipeline, which we will detail here.
+
+```mermaid
+graph TD
+fx_code(fa:fa-cog Firefox code) --> firefox(fa:fa-firefox Firefox Telemetry)
+fx_extensions(fa:fa-cog Mozilla extensions) --> firefox
+fx_hybrid(fa:fa-cog Hybrid Content) --> firefox
+mobile(fa:fa-cog Mobile products) --> mobile_telemetry(fa:fa-firefox Mobile Telemetry)
+firefox -->|main ping| pipeline((fa:fa-database Firefox Data Pipeline))
+firefox -.->|events ping, planned| pipeline
+mobile_telemetry --> |mobile events ping| pipeline
+pipeline -->|Firefox events| main_summary[fa:fa-bars main summary table]
+main_summary --> events_table[fa:fa-bars events table]
+pipeline -->|Mobile events| focus_events_table[fa:fa-bars mobile events table]
+main_summary --> redash(fa:fa-bar-chart Redash)
+events_table --> redash
+events_table -.->|planned| amplitude(fa:fa-bar-chart Amplitude)
+focus_events_table --> redash
```

Comment:
I would change this to `mobile_events`, here and everywhere else. While technically we are sending events to Amplitude from `focus-events`, we also don't want anyone else to use that ping, and will eventually transition focus to `mobile-events` as well.